File: frontend/Web/src/app/_shared/models/_auth/login.response.ts
import { Paciente } from "../pessoa/paciente/paciente";
import { Tecnico } from "../pessoa/tecnico/tecnico";

export interface LoginResponse {
    success: boolean;
    user: Tecnico | Paciente | null;
    tipo?: 'tecnico' | 'paciente';
    token?: string;
    message?: string;
}

File: frontend/mobile/app/src/main/java/com/tecno/aging/domain/models/auth/LoginResponse.kt
package com.tecno.aging.domain.models.auth

data class LoginResponse(
    val Perfil: String? = null,
    val ID: String? = null,
    val Nome: String? = null,
    val message: String? = null,
    val token: String? = null,
    val success: Boolean
)